README.md

```diff
@@ -1,52 +1,59 @@
 # notifications-antivirus
 
-GOV.UK Notify Antivirus service. Read and write scan jobs via a scan queue.  Retrieves the supplied filename from the scan S3 bucket and uses ClamAV to scan the file. Sends the scan status back via a queue to update the notification status.
+Reads jobs from a queue, using the supplied filename to fetch files from an S3 bucket, and ClamAV to scan them. Sends the scan status back by creating a new job to update the notification status.
 
-##  Environment Variables
+## Setting up
 
-Creating the environment.sh file. Replace [unique-to-environment] with your something unique to the environment. Your AWS credentials should be set up for notify-tools (the development/CI AWS account).
+### Docker container
 
-Create a local environment.sh file containing the following:
+This app uses dependencies that are difficult to install locally. In order to make local development easy, we run app commands through a Docker container. Run the following to set this up:
 
+```shell
+  make bootstrap
 ```
-echo "
 
-export NOTIFICATION_QUEUE_PREFIX='YOUR_OWN_PREFIX'
-export NOTIFY_ENVIRONMENT='development'
-export FLASK_APP=application.py
-export FLASK_DEBUG=1
-export WERKZEUG_DEBUG_PIN=off
-export AWS_ACCESS_KEY_ID='YOUR_TOOLS_AWS_ACCESS_KEY'
-export AWS_SECRET_ACCESS_KEY='YOUR_TOOLS_AWS_SECRET_KEY'
-export NOTIFY_LOG_PATH='/var/log/notify/antivirus'
+Because the container caches things like Python packages, you will need to run this again if you change things like "requirements.txt".
+
+### `environment.sh`
+
+In the root directory of the application, run:
 
+```
+echo "
+export NOTIFICATION_QUEUE_PREFIX='YOUR_OWN_PREFIX'
 "> environment.sh
 ```
 
-NOTES:
+Things to change:
 
- * Replace the placeholder key and prefix values as appropriate
- * The unique prefix for the queue names prevents clashing with others' queues in shared amazon environment and enables filtering by queue name in the SQS interface.
+- Replace YOUR_OWN_PREFIX with local_dev_\<first name\>.
```

Comment:
Most people just use their first name as the queue prefix. Maybe this could just mention that the value of `NOTIFICATION_QUEUE_PREFIX` needs to match what you're using for other apps

Reply:
That's true. I've not done this in other README, but I agree it makes sense to mention it.

I've amended the "Tweak README" commit to mention it 👍.